laser_scan_matcher/src/laser_scan_matcher.cpp

```diff
@@ -162,6 +166,12 @@ void LaserScanMatcher::initParams()
   if (!nh_private_.getParam ("use_vel", use_vel_))
     use_vel_ = false;
 
+  // **** Are velocity input messages stamped?
+  // if false, will subscribe to Twist msgs on /vel
+  // if true, will subscribe to TwistStamped msgs on /vel
+  if (!nh_private_.getParam ("stamped_vel", stamped_vel_))
```

Comment:
Ok, so the launch file is what the users of this package makes by themselves, right? If that's the case, this needs documented somewhere (maybe adding to README or equivalent?) otherwise no one notices this feature even if this feature might be valuable to them.